android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -206,17 +189,18 @@ public void fetchRelease(PluginCall call) {
 
     @PluginMethod
     public void captureEnvelope(PluginCall call) {
-        String envelope = call.getString("envelope");
         try {
-            File installation =  new File(sentryOptions.getOutboxPath(), UUID.randomUUID().toString());
+            String envelope = call.getString("envelope");
+            final String outboxPath = HubAdapter.getInstance().getOptions().getOutboxPath();
```

Comment:
you need to check if `outboxPath` isnt null or empty and bail out if so otherwise it'll crash